docs/product-getting-started-guide/src/main/asciidoc/deploying-application-proc.adoc

```diff
@@ -1,37 +1,22 @@
 [id='deploy-app-process']
-= {PRODUCT} Application Deployment
+= Deploying the application
 
-When business analysts create processes and rules, deploying changes can be done with a click of a button. {PRODUCT} is the leading platform in integrating with other systems. With {PRODUCT}, developers can do the following.
-
-* Integrate {PRODUCT} with the latest software solutions in the business.
-* Modify or add {PRODUCT} functionality.
-* Leverage provided tools to build their solutions faster.
-
-{PRODUCT} focuses on as fast deployment time as possible. Because business processes and decisions often change quickly, {PRODUCT} enables quick reaction time.
-
-Red Hat also offers a range of fully supported solutions which can be integrated with {PRODUCT}. An example of such integration is deploying {PRODUCT} on the Red Hat Openshift Container Platform, or integrating {PRODUCT} with Red Hat JBoss Fuse.
-
-The following chapter instructs you how to deploy an application in {PRODUCT}. You can deploy:
-
-* A custom application.
-* The getting started application.
-
-To import the getting started application, follow <<creating_business_project>>.
+The following chapter instructs you how to build and deploy a new instance of the *Mortgage Process* application in {PRODUCT}.
```

Comment:
I'm not sure what's happening here, because what you've undone seems like what I'm about to suggest to do. Chapters 7 - 9 are choppy and inconsistent with the structure of the rest of the doc. It's an easy fix, imo:

For each chapter, just make the the top level the "Doing x" with what you already have as the context paragrph, and then the .Procedure segment. Each would be just one section. Example, make "Chapter 7. Deploy" just "Chapter 7. Deploying the application". Use the same content you already have as the context, then go into the .Procedure segment. Do the same for executing and monitoring sections.

This is what the sections should look like:

ch. 7 Deploying the Mortgage Process application 
<context info you already have>
.Prereqs
<item(s)>
.Procedure
<intro sentence>
<steps>

ch. 8 Executing the Mortgage Process application 
<context info you already have>
.Prereqs
<item(s)>
.Procedure
<intro sentence>
<steps>

ch. 9 Monitoring the Mortgage Process application 
<context info you already have>
.Prereqs
<item(s)>
.Procedure
<intro sentence>
<steps>

If you wanted to have a general "7 Deploying process" section followed by "7.1 Deploying the Mortgage ..." (and same for the other two chapters) to go from general to specific, you could do that too. But three levels is just too much and unnecessary.